hera_cal/tests/conftest.py

```diff
@@ -14,15 +14,18 @@
 
 @pytest.fixture(autouse=True, scope="session")
 def setup_and_teardown_package():
-    # Try to download the lateest IERS table. If the download succeeds, run a
-    # computation that requires the values, so they are cached for all future
-    # tests. If it fails, turn off auto downloading for the tests and turn it
-    # back on in teardown_package (done by extending auto_max_age).
+    """Make data/test directory to put test output files in."""
```

Comment:
Is this accurate? It doesn't look like you're making a test directory. I think it's copied from `pyuvdata`, which isn't doing that anymore either, so it should be fixed there too.

Reply:
No, it's not. I removed it.